Assets/HotmaxEditorVR/stateManager.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class stateManager : MonoBehaviour
+{
+    public static editorModes editorMode = editorModes.universalTransformMode;
+    public enum editorModes
+    {
+        universalTransformMode,
+        cloneDeleteMode,
+        openMenuMode
+    }
+    public delegate void editorModeHandler(editorModes value);
+    public static event editorModeHandler editorModeEvent;
+
+
+    public static rotationModes rotationMode;
+    public enum rotationModes
+    {
+        xRotationMode,
+        yRotationMode,
+        zRotationMode
+    }
+    public delegate void rotationModeHandler(rotationModes value);
+    public static event rotationModeHandler rotationModeEvent;
+
+
+
+
```

Comment:
clean up spacing, make sure vars are with their delegate and event without break